accountServices.py

```diff
@@ -113,7 +113,7 @@ def connectLDAP(self):
         return conn
 
     def chPassword(self, username, password):
-        return self.kadmin.chPassword(username, password)
+        return self.kadm.get_princ(username).change_password(password)
```

Comment:
This should probably be split into 2 calls with a try/except.  If the get_princ() call fails you'll be trying to call a method on `None`.